src/main/java/org/candlepin/subscriptions/metering/job/OpenShiftMeteringJob.java

```diff
@@ -59,13 +59,17 @@ public OpenShiftMeteringJob(PrometheusMetricsTaskManager tasks, ApplicationClock
     public void run() {
         Duration latency = appProps.getPrometheusLatencyDuration();
         int range = metricProperties.getOpenshift().getRangeInMinutes();
-        OffsetDateTime startDate = clock.startOfHour(clock.now().minus(latency).minusMinutes(range));
+        int offset = metricProperties.getOpenshift().getOffsetInMinutes();
+
+        OffsetDateTime startDate = clock.startOfHour(clock.now().minus(latency)
+            .minusMinutes(range).minusMinutes(offset));
```

Comment:
This is precisely the kind of change we need, but we need to apply it instead in `org.candlepin.subscriptions.tally.job.CaptureSnapshotsTaskManager#updateHourlySnapshotsForAllAccounts`

Rationale follows;

Given:
- current `develop` version of the code
- a running instance of `OpenShiftMeteringJob` at ~1am UTC.
- a running instance of `CaptureMetricsSnapshotTask` (scheduled by `CaptureSnapshotsTaskManager` at ~1am UTC)
- default configurations

then:
- the range for the OpenShiftMeteringJob would be (`1am - 4 hours - 1 hour` through `1am - 4 hours`) -> (8pm to 9pm)
- the range for the CaptureMetricsSnapshotTask would be (`1 am - 4 hours - 24 hours` through `1am - 4 hours`) -> (9pm to 9pm)

In other words, CaptureMetricsSnapshotTask will currently operate on an overlapping hour with the metering job.

Please let me know if the above makes sense and is accurate (some smoketesting might be helpful).

Reply:
That makes sense.  I will add this to the card.  PR forthcoming.